app.py

```diff
@@ -11,8 +11,8 @@ def index():
 
 @app.get('/movies')
 def list_all_movies():
-    # TODO: Feature 1
-    return render_template('list_all_movies.html', list_movies_active=True)
+    movies = movie_repository_singleton.get_all_movies()
+    return render_template('list_all_movies.html', list_movies_active=True, movieList=movies)
```

Comment:
I recommend using snake_case for `movieList` so it's consistent with the rest of the code.